README.md

```diff
@@ -445,6 +448,145 @@ No, see the [cited document](https://eclass.eu/fileadmin/Redaktion/pdf-Dateien/B
 A: In the current API specification AAS Part 2 V1.0RC03 the unique Submodel Id is expected for a GET/submodels/{submodelIdentifier} and no longer the Submodel IdShort. This enables distinguishing between the particular submodel instance.
 
  (Answered 2022-11-21)
+
+
+ **[What are the right attribute values for 'Descriptor/endpoints'?](#id47)** <a id="id47"></a>
+
+A: The `interface` attribute shall contain the Service Specification identifier together with a version tag. The `protocolInformation` object contains additional information as for instance the intended root endpoint as the URL (`href`) and how to reach it. `endpointProtocol` states which protocol shall be used. It is intended for clients to select the proper socket library without needing to parse the `href`content. 
+
+The `endpointProtocolVersion` contains the major.minor version of the `endpointProtocol` entry. For HTTP, versions `1.0`, `1.1`, or `2.0` are possible entries.
```

Comment:
Perhaps mention that more than one version can be supported since it is an array?